plugins/Cache.cpp

```diff
@@ -209,6 +210,10 @@ bool BedrockPlugin_Cache::processCommand(SQLite& db, BedrockCommand& command) {
         //
         verifyAttributeSize(request, "name", 1, MAX_SIZE_SMALL);
         const string& valueHeader = request["value"];
+        const string& name = request["name"];
+        command.crashIdentifyingValues.insert("name");
+        command.crashIdentifyingValues.insert("value");
```

Comment:
Why not value in peek?

Reply:
peek doesn't have a value param, all peek does is take in a name and return a value